app/models/game.rb

```diff
@@ -44,4 +46,17 @@ def piece_at(x, y)
   def is_full?
     white_user_id && black_user_id
   end
+
+  def first_move
```

Comment:
This is assigning a variable called player_turn to 'white' but isn't saving that to the DB. You probably should use something like update_attributes(:player_turn => 'white').